statsproduct.php

```diff
@@ -157,6 +157,7 @@ public function hookAdminStatsModules()
     {
         $id_category = (int)Tools::getValue('id_category');
         $currency = Context::getContext()->currency;
+	$id_lang = $this->context->language->id;
```

Comment:
Don't mix tabs and spaces please :)

Reply:
Fixed it in https://github.com/PrestaShop/statsproduct/pull/8/commits/36e97bc9ce25ad3c20621340d37c0b53c9deea1a